app/src/main/java/com/mikhailovskii/weatherandroid/util/Constants.kt

```diff
@@ -0,0 +1,15 @@
+package com.mikhailovskii.weatherandroid.util
+
+const val HALF_SCREEN = 0.5
+
+const val QUARTER_SCREEN = 0.25
+
+const val BASE_MARGIN = 16
+
+const val PERMISSION_CODE = 100
+
+const val DIFFERENCE_KELVIN_CELSIUS = 273
```

Comment:
@SergeiMikhailovskii better name would be `KELVIN_TO_CELSIUS_COEFFICIENT` or `KELVIN_TO_CELSIUS_COEF` (or `K_TO_C_COEF` if it belongs to some class like `TemperatureUtil` that also knows how to convert temperature from one scale to another)